pom.xml

```diff
@@ -7,10 +7,11 @@
   <version>1.11.2-SNAPSHOT</version>
 
   <properties>
-    <jackson.version>2.6.6</jackson.version>
+    <jackson.version>2.9.0</jackson.version>
```

Comment:
there a reason you aren't going to the latest? incompatibility with ES?

Reply:
I thought so, but it was apparently i was missing the dataformatter, so I'm not sure anymore. I'll test out latest when i get a second

Reply:
tests says its working fine, so it was just the extra library needed.